tools/build.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const commandRunner = require('./run-command');
+
+const build = 'npm run build';
+const build_db = 'npm run build -- --public-url=/"$GH-PAGES/"';
+
+var mkdirp = require('mkdirp');
+var rimraf = require('rimraf');
+var copy = require('copy');
+
+rimraf('./dist', {}, function() {});
+
+mkdirp('dist', function(err) {
```

Comment:
This is an async function. We should either use a [sync](https://www.npmjs.com/package/mkdirp#mkdirpsyncdir-opts) version or promisify the async version and link it with the rest of the chain.